server/src/api/modules/competitions/competition.service.js

```diff
@@ -600,6 +586,33 @@ async function getParticipants(id) {
   return participants;
 }
 
+/**
+ * Get outdated participants for a specific competition id.
+ * A participant gets outdated after 10 mins
```

Comment:
Can you clarify this? Something like "A participant is considered outdated 10 minutes after their last update"